### PR TITLE
feat(data-table): row actions

### PR DESCRIPTION
**Pull request by chasestarr**
#### Description

Screen Shot 2019-10-17 at 1 27 06 PM

#### Scope

- [x] Minor: New Feature

### CODE REVIEW

File: src/data-table/measure-column-widths.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ type MeasureColumnWidthsPropsT = {
 };
 
 // sample size could likely be generated based on row count, to have higher confidence
-const MAX_SAMPLE_SIZE = 10;
+const MAX_SAMPLE_SIZE = 50;
```
Comment (chasestarr) [inline src/data-table/measure-column-widths.js]: I changed this to handle the variable movie title lengths in a new example.. Signals to me that it needs to be configurable. Will create a new PR to do so

**Comment by gergelyke** [comment]
Looks great! one thing I found is that when you select a row, the actions are not positioned correctly:

Screen Shot 2019-10-18 at 3 56 39 PM

**Comment by chasestarr** [comment]
> Looks great! one thing I found is that when you select a row, the actions are not positioned correctly:

good catch!

**Comment by chasestarr** [comment]
refactored the code so that overlay position is unrelated to y offset from parent container

**Pull request merged**